README.md

```diff
@@ -27,7 +27,8 @@ Some more guidelines:
 This file is written in the [Markdown format](https://guides.github.com/features/mastering-markdown/).
 
 Lines starting with `*` are automatically rendered as bullet points and phrases containing `[name](URL)` are automatically hyperlinked.
-<!--ALPHA-->
```

Comment:
Please don't remove this tag. All the names should be between these tags for travis check to work.